jerry-core/ecma/builtin-objects/ecma-builtins.inc.h

```diff
@@ -301,7 +301,7 @@ BUILTIN (ECMA_BUILTIN_ID_COMPACT_PROFILE_ERROR,
 /* The Global object (15.1) */
 BUILTIN (ECMA_BUILTIN_ID_GLOBAL,
          ECMA_OBJECT_TYPE_GENERAL,
-         ECMA_BUILTIN_ID__COUNT /* no prototype */,
+         ECMA_BUILTIN_ID_OBJECT_PROTOTYPE,
```

Comment:
Ok, I'll add a comment.

Please see the https://github.com/tc39/test262/blob/es5-tests/test/suite/ch15/15.2/15.2.3/15.2.3.14/15.2.3.14-6-6.js testcase's 17 line, where it calls the 'hasOwnProperty' on the global object.